jekyll/_layouts/classic-docs.html

```diff
@@ -41,6 +41,25 @@
               {% endfor %}
             </div>
             {% endif %}
+
+            {% if page.page-type != "index" and page.page-type != "homepage" %}
+            <time id="postedon" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
+              <div id="postedon-text">
+                <div id="postedon-svg">
+                  <svg width="18" height="18" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is there anyway we could move this into an SVG asset file and just reference it? I'm thinking mainly so that it's accessible to imagemin. also this seems like a compass item and it would be nice to group these icons together.

Reply:
I can add it to asset. Not sure how to get is as a compass item. Can you assist?